### PR TITLE
Added New Game: Ping pong

### LINKED ISSUE #5034: [New game]: Pong
### 🎮 Game Request

Game logic and basic description: The game will be a basic ping pong game where both players will control paddles using movement keys and try to prevent the ball from going into their area and score a point.

### Point down the features

Game points
1) A Playing area with a ball
2) The ball will be bouncing between the players
3) The players have to defend their goals with their paddles.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by AaryanManghnani**
## PR Description 📜

A Class Ping Pong Game in which both players control their paddles and defend their side while trying to score on their opponent.
Fixes #5034

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in GameZone's README.md
- [x] I have added README.md in my Game folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.
- [x] I have added my Game to gamesData.json file and it is loading properly on main website as intended.

<hr>

## Add your screenshots(Optional) 📸
image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @AaryanManghnani ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -822,7 +822,7 @@ This repository also provides one such platforms where contributers come over an
 | [Cosmic_Blast](https://github.com/kunjgit/GameZone/tree/main/Games/Cosmic_Blast) |
 |[Mole](https://github.com/taneeshaa15/GameZone/tree/main/Games/Mole)|
 |[Pottery-Game](https://github.com/kunjgit/GameZone/tree/main/Games/Pottery-Game)|
-
+|[Ping_Pong](https://github.com/AaryanManghnani/GameZone/tree/Ping_Pong/Games/Ping_Pong)
```
Comment (ishita-43) [inline README.md:825]: Chane name of the repo to kunjgit instead of your repo name

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make following update

File: README.md (hunk 1 above)
Comment (ishita-43) [reply README.md:825]: > refer other entries for the format

**Review by ishita-43 — APPROVED** [review]
Pr is ready to merge

**Comment by kunjgit** [comment]
Thank you  @AaryanManghnani , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**